content/_index.md

```diff
@@ -25,23 +25,41 @@ OSM runs an Envoy based control plane on Kubernetes, can be configured with SMI
 
 ## Install
 
+![OSM Install Demo](https://github.com/openservicemesh/osm/raw/release-v0.8/img/osm-install-demo-v0.2.0.gif "OSM Install Demo")
+
 ### Prerequisites
-- Kubernetes cluster running Kubernetes v1.18.0 or greater
-- kubectl current context is configured for the target cluster install
-  - ```kubectl config current-context```
+- Kubernetes cluster running Kubernetes v1.18.0 or greater.
+- `kubectl` installed and configured to connect to your Kubernetes cluster.
+  - To display the cluster `kubectl` is configured to connect to, use `kubectl config current-context`
 
-### Get the OSM CLI
+### Install the OSM CLI
 
-The simplest way of installing Open Service Mesh on a Kubernetes cluster is by using the `osm` CLI.
+The `osm` CLI is the simplest way to install OSM on your Kubernetes cluster. To install the `osm` CLI, download and unpack the `osm` binary from the [Releases page](https://github.com/openservicemesh/osm/releases) and add it to `$PATH`. For more details on installing the `osm` CLI, see [Installation](https://docs.openservicemesh.io/docs/install/).
```

Comment:
I am wondering if we might want to break all this install info out to the install page. I think we run the risk of desynced info between here and https://github.com/openservicemesh/osm-docs/blob/main/content/docs/install/_index.md.

Reply:
Discussed elsewhere; we'll use this info here for now as a pointer to elsewhere, and see if we run into issues.